tdvt_workspace/README.md

```diff
@@ -3,19 +3,12 @@ Instructions for running TDVT testing on the Tableau Databricks connector:
 1. Install TDVT on your Windows machine using the guide from https://tableau.github.io/connector-plugin-sdk/docs/tdvt. The resulting tdvt_workspace directory should contain the following subdirectories: config, plugins, and tds.
 
 2. Prepare the testing tables.
-- Set `spark.sql.crossJoin.enabled true` in the cluster Spark configuration tab
-- Upload the TestV1 datasets from
-   https://github.com/tableau/connector-plugin-sdk/tree/master/tests/datasets/TestV1
-from the `Data -> Add Data` menu in your Databricks environment.
-- Run the `testv1_datasets` notebook inside the notebooks directory to
-   create the testing tables.
+Use the tables in dogfood(https://e2-dogfood.staging.cloud.databricks.com/sql/explore/data/tableau/tableau_tdvt?o=6051921418418893) which is under Catalog `tableau` Schema `tableau_tdvt`
```

Comment:
You are dropping here some steps on how to upload data to a new env. Also, note that this repo is public.

Reply:
I took Bogdan's comments that we would eradicate or obscure the workspace etc internal IDs and/or even the dogfood URL.